snappi_ixnetwork/trafficitem.py

```diff
@@ -422,26 +477,25 @@ def _configure_pattern(self,
                              ValueType='valueList',
                              ValueList=pattern.values)
         elif pattern.choice == 'increment':
-            # try:
-            ixn_field.update(Auto=False,
-                            ValueType='increment',
-                            ActiveFieldChoice=field_choice,
-                            StartValue=pattern.increment.start,
-                            StepValue=pattern.increment.step,
-                            CountValue=pattern.increment.count)
-            # except Exception as e:
-            #     import pdb; pdb.set_trace()
-            #     print(e)
+            try:
+                ixn_field.update(Auto=False,
+                                 ValueType='increment',
+                                 ActiveFieldChoice=field_choice,
+                                 StartValue=pattern.increment.start,
+                                 StepValue=pattern.increment.step,
+                                 CountValue=pattern.increment.count)
+            except Exception as e:
+                print(e)
         elif pattern.choice == 'decrement':
-            # try:
-            ixn_field.update(Auto=False,
-                            ValueType='decrement',
-                            ActiveFieldChoice=field_choice,
-                            StartValue=pattern.decrement.start,
-                            StepValue=pattern.decrement.step,
-                            CountValue=pattern.decrement.count)
-            # except Exception as e:
-            #     print(e)
+            try:
+                ixn_field.update(Auto=False,
```

Comment:
`test_errors.py` test is failing in CI/CD as try, except block is bypassing the server side error.

Reply:
fix in https://github.com/open-traffic-generator/ixnetwork/commit/04c5079dbc50aa2d3eecfc840ab95371e0298f65